docs/integrations/destinations/snowflake.md

```diff
@@ -106,14 +112,14 @@ commit;
 
 You should now have all the requirements needed to configure Snowflake as a destination in the UI. You'll need the following information to configure the Snowflake destination:
 
-* **Host**
-* **Role**
-* **Warehouse**
-* **Database**
-* **Schema**
-* **Username**
-* **Password**
-* **JDBC URL Params** (Optional)
+* **Host** : The host domain of the snowflake instance (must include the account, region, cloud environment, and end with snowflakecomputing.com). Example - `accountname.us-east-2.aws.snowflakecomputing.com`
```

Comment:
For this section of descriptions, are external docs from Snowflake that can/should be referenced?

For example, with "Role", this seems like it's appropriate: https://docs.snowflake.com/en/user-guide/security-access-control-overview.html#roles.